tests/Meziantou.Analyzer.Test/Rules/ClassWithEqualsTShouldImplementIEquatableTAnalyzerTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System.Threading.Tasks;
```

Comment:
It would be nice to introduce a new test with nullable reference types. There is no example in this repository but I think you can use something like:

````csharp
            var originalCode = @"
#nullable enable
class [|Test|]
{
    public bool Equals(Test? other) => throw null;
}";
            var modifiedCode = @"
class Test : System.IEquatable<Test> // I don't know what's best here IEquatable<Test> or IEquatable<Test?> (I think you can check what VS generates for equality members)
{
    public bool Equals(Test? other) => throw null;
}";
````